plugins/org.eclipse.embedcdt.debug.gdbjtag.ui/src/org/eclipse/embedcdt/internal/debug/gdbjtag/ui/preferences/PeripheralRendering.java

```diff
@@ -228,6 +228,8 @@ public void menuAboutToShow(IMenuManager manager) {
 		addDebugEventListener();
 		addToSyncService();
 		trackTreeSelectionChanges();
+		// Refresh the UI, since while initialising the controls it would have missed some debug events
```

Comment:
@jonahgraham, what is the current max line length for CDT? Does this comment line comply?

Reply:
@ilg-ul I just splitted that into two lines.

Reply:
Ok, thank you.

Reply:
Thanks @vishnusarathashling for splitting it.

IIRC we try to keep < 120 chars for everything, including comments.